packages/tokens/src/color-primitives.yaml

```diff
@@ -189,3 +189,14 @@ color:
         value: hsl(45.1, 57.4%, 38.6%)
       900:
         value: hsl(45, 57.8%, 32.5%)
+  gradient:
```

Comment:
If possible, can we also have these render in the [global tokens story](https://github.com/launchdarkly/launchpad-ui/blob/main/packages/tokens/stories/tokens.stories.tsx#L58-L65)?